python/lsst/afw/cameraGeom/cameraFactory.py

```diff
@@ -87,7 +87,7 @@ def makeCameraFromPath(cameraConfig, ampInfoPath, shortNameFunc):
     @return camera (an lsst.afw.cameraGeom.Camera)
     """
     ampInfoCatDict = dict()
-    for detectorConfig in cameraConfig.detectorList.itervalues():
+    for detectorConfig in cameraConfig.detectorList.values():
```

Comment:
I didn't name these attributes :-)  Regarding dict efficiency, you may be right. `futurize` does not do that automatically.